src/ucp/api/ucp.h

```diff
@@ -335,26 +335,37 @@ enum ucp_ep_close_mode {
  * @ingroup UCP_ENDPOINT
  * @brief UCP performance fields and flags
  *
- * The enumeration allows specifying which fields in @ref ucp_ep_perf_attr_t are
+ * The enumeration allows specifying which fields in @ref ucp_ep_evaluate_perf_request_t are
  * present and operation flags are used. It is used to enable backward
  * compatibility support.
  */
-typedef enum ucp_ep_perf_attr_field {
-    /** Enables @ref ucp_ep_perf_attr_t::message_size */
-    UCP_EP_PERF_ATTR_FIELD_MESSAGE_SIZE       = UCS_BIT(0),
+typedef enum ucp_ep_perf_request_field {
```

Comment:
for consistency, it should be
ucp_ep_perf_param_field - for input
ucp_ep_perf_attr_field - for output